celery/apps/beat.py

```diff
@@ -71,25 +76,35 @@ def __init__(self, max_interval=None, app=None,
         if not isinstance(self.loglevel, numbers.Integral):
             self.loglevel = LOG_LEVELS[self.loglevel.upper()]
 
-    def run(self):
+    @property
```

Comment:
Using this pattern to accommodate the required type safety from MyPy (obviously this doesn't really provide type safety). Would you prefer that I refactor the class to ensure actual type safety?

Reply:
This appears to be out of scope so you can leave it like that for now.
Maybe in a different PR if you wish.

Nice idea though!

Reply:
Sounds good!